ekumamait/xcnt-backend
# #170544968 setup mongodb connection

**What does this PR do?**
	- Add connection file
	- Add environment variables
	- Add models and helpers

**Description of Task to be completed**
This task setups a Mongo Db connection to save the expenses from the event stream and allows the client to fetch all, a single one by `ID` and update.

**How should this be manually tested?**
* Switch to the branch `ft-db-setup-#170544968`
* Start the server using `npm start`
* Start `Postman` app and run this endpoint to see saved event streams

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
#170544968

**Screenshots (if appropriate)**
All Expenses:
<img width="1070" alt="Screen Shot 2020-01-07 at 11 26 35" src="https://user-images.githubusercontent.com/9946845/71880125-aee2f980-3140-11ea-8bf1-d7ad0fe8b66e.png">

Single Expense:
<img width="1150" alt="Screen Shot 2020-01-07 at 11 26 11" src="https://user-images.githubusercontent.com/9946845/71880166-c28e6000-3140-11ea-9b22-b30f77b13cab.png">

Updated Expense:
<img width="1067" alt="Screen Shot 2020-01-07 at 11 30 22" src="https://user-images.githubusercontent.com/9946845/71880357-2add4180-3141-11ea-9d2d-15690198215d.png">
